analysis_runner/cromwell.py

```diff
@@ -629,15 +664,3 @@ def _copy_resource_group_into_batch(
         )
 
     return output_filename
-
-
-def get_cromwell_oauth_token():
```

Comment:
Would removing this entirely cause an issue in server/cromwell.py? 
https://github.com/populationgenomics/analysis-runner/runs/6398752286?check_suite_focus=true#step:5:43

Reply:
Good catch!

Reply:
Thanks :) Fixed